.github/workflows/BFS.yaml

```diff
@@ -0,0 +1,235 @@
+# Copyright (c) 2023, Oracle Corporation and/or its affiliates.
+#
+# Licensed under the Universal Permissive License v 1.0 as shown at
+# http://oss.oracle.com/licenses/upl.
+
+# ---------------------------------------------------------------------------
+# Verrazzano GitHub Actions Keycloak BFS.
+# ---------------------------------------------------------------------------
+
+name: Keycloak BFS
+run-name: GitHub Actions workflow triggered by changes to Keycloak BFS 🔑
+
+on:
+  push:
+    branches:
+      - 'oracle/release/*'
+      - 'bfs/*'
+  workflow_dispatch:
+
+env:
+  REGISTRY: ghcr.io
+  NAMESPACE: verrazzano
+
+concurrency:
+  # Only run once for latest commit per ref and cancel other (previous) runs.
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+      # This is used to complete the identity challenge
+      # with sigstore/fulcio when running outside of PRs.
+      id-token: write
+
+    steps:
+      # Get Keycloak version
+      - name: Get Keycloak version
+        run: echo "KEYCLOAK_VERSION=$(echo ${{ github.ref_name }} | sed 's:.*/::')" >> "$GITHUB_ENV"
```

Comment:
Will this work for BFS branch names? I don't think those will have a short name matching the keycloak version

Reply:
Branches must follow the naming convention bfs/***/KEYCLOAK_VERSION

Reply:
Need branch name validation

Reply:
examples include:
-  bfs/example/example/example/KEYCLOAK_VERSION
- bfs/example/KEYCLOAK_VERSION